aimpoint_mon/update_observed_aimpoints.py

```diff
@@ -352,11 +352,11 @@ def plot_observed_aimpoints(obs_aimpoints):
                 plot_cxctime(times[nok], obs_aimpoints[axis][nok], marker='*',
                              markerfacecolor=c, **kwargs)
             if np.any(lolims[axis]):
-                plt.errorbar(DateTime(times[lolims[axis]]).plotdate,
```

Comment:
`DateTime.plotdate` gives the wrong answer for modern matplotlib because of a change in mpl.